sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1600,6 +1600,114 @@ def _df_getitem_bool_array_idx_impl(self, idx):
 
 @sdc_overload(operator.getitem)
 def sdc_pandas_dataframe_getitem(self, idx):
+    """
+        Intel Scalable Dataframe Compiler User Guide
+        ********************************************
+        Pandas API: pandas.DataFrame.getitem
+
+        Get column(s) of DataFrame by key.
```

Comment:
This is not accurate - examples show that df[1:3] selects rows from the df, not columns. Maybe:
```suggestion
        Get data from a DataFrame by indexer.
```